slack-api-client/src/main/java/com/slack/api/rate_limits/metrics/impl/BaseMemoryMetricsDatastore.java

```diff
@@ -264,13 +264,6 @@ public void updateCurrentQueueSize(String executorName, String teamId, String me
     @Override
     public void setCurrentQueueSize(String executorName, String teamId, String methodName, Integer size) {
         if (this.isStatsEnabled()) {
-            CopyOnWriteArrayList<String> messageIds = getOrCreateMessageIds(executorName, teamId, methodName);
```

Comment:
These lines of code are completely unnecessary. They are just an overhead when executing this method.